commandline/src/main/java/tech/pegasys/web3signer/commandline/PicoCliAwsSecretsManagerParameters.java

```diff
@@ -0,0 +1,150 @@
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Option;
+
+public class PicoCliAwsSecretsManagerParameters implements AwsSecretsManagerParameters {
+  public static final String AWS_SECRETS_ENABLED_OPTION = "--aws-secrets-enabled";
+  public static final String AWS_SECRETS_AUTH_MODE_OPTION = "--aws-secrets-auth-mode";
+  public static final String AWS_SECRETS_ACCESS_KEY_ID_OPTION = "--aws-secrets-access-key-id";
+  public static final String AWS_SECRETS_SECRET_ACCESS_KEY_OPTION =
+      "--aws-secrets-secret-access-key";
+  public static final String AWS_SECRETS_REGION_OPTION = "--aws-secrets-region";
+  public static final String AWS_SECRETS_PREFIXES_FILTER_OPTION = "--aws-secrets-prefixes-filter";
+  public static final String AWS_SECRETS_TAG_NAMES_FILTER_OPTION = "--aws-secrets-tag-names-filter";
+  public static final String AWS_SECRETS_TAG_VALUES_FILTER_OPTION =
+      "--aws-secrets-tag-values-filter";
+  public static final String AWS_CONNECTION_CACHE_SIZE_OPTION = "--aws-connection-cache-size";
+
+  @Option(
+      names = AWS_SECRETS_ENABLED_OPTION,
+      description =
+          "Set to true to enable bulk loading from the AWS Secrets Manager service."
+              + " (Default: ${DEFAULT-VALUE})",
+      paramLabel = "<BOOL>")
+  private boolean awsSecretsManagerEnabled = false;
+
+  @Option(
+      names = AWS_SECRETS_AUTH_MODE_OPTION,
+      description =
+          "Authentication mode for AWS Secrets Manager service. Valid Values: [${COMPLETION-CANDIDATES}]"
+              + " (Default: ${DEFAULT-VALUE})",
+      paramLabel = "<AUTHENTICATION_MODE>")
+  private AwsAuthenticationMode authenticationMode = AwsAuthenticationMode.SPECIFIED;
+
+  @Option(
+      names = {AWS_SECRETS_ACCESS_KEY_ID_OPTION},
+      description =
+          "AWS Access Key Id to authenticate Aws Secrets Manager. Required for SPECIFIED authentication mode.",
+      paramLabel = "<ACCESS_KEY_ID>")
+  private String accessKeyId;
+
+  @Option(
+      names = {AWS_SECRETS_SECRET_ACCESS_KEY_OPTION},
+      description =
+          "AWS Secret Access Key to authenticate Aws Secrets Manager. Required for SPECIFIED authentication mode.",
+      paramLabel = "<SECRET_ACCESS_KEY>")
+  private String secretAccessKey;
+
+  @Option(
+      names = {AWS_SECRETS_REGION_OPTION},
+      description =
+          "AWS region where Secrets Manager service is available. Required for SPECIFIED authentication mode.",
+      paramLabel = "<Region>")
+  private String region;
+
+  @Option(
+      names = AWS_SECRETS_PREFIXES_FILTER_OPTION,
+      description =
+          "Optional comma-separated list of secret name prefixes filter to apply while fetching secrets from AWS secrets manager."
+              + " Applied as AND operation with other filters. (Default: ${DEFAULT-VALUE})",
+      split = ",")
+  private List<String> prefixesFilter = Collections.emptyList();
```

Comment:
Think this should just be prefixFilters not prefixesFilters. We have the plural on filters for other filters.